bulbs/styles/static/styles/stylesheets/global/breakpoints.scss

```diff
@@ -0,0 +1,43 @@
+///////////////////////////////
+// ************************* //
+//         BREAKPOINTS       //
+// ************************* //
+///////////////////////////////
+
+// 480px 30em
+// 600px 37.5em
+// 768px 48em
+// 992px 62em
+// 1382px 86.375em
+
+$bp2 : 480px;
```

Comment:
Are there more verbose names we can use for these other than just numbered ... do they match to any devices/device categories